--- a/net/mac80211/tx.c	2020-06-15 20:03:27.234467031 +0100
+++ b/net/mac80211/tx.c	2020-06-16 00:21:15.171394185 +0100
@@ -164,7 +164,7 @@
 			if (r->flags & IEEE80211_RATE_MANDATORY_A)
 				mrate = r->bitrate;
 			break;
-		case IEEE80211_BAND_60GHZ:
+		case NL80211_BAND_60GHZ:
 			/* TODO, for now fall through */
 		case NUM_NL80211_BANDS:
 			WARN_ON(1);

--- a/net/mac80211/mlme.c	2020-06-15 23:58:26.431440026 +0100
+++ b/net/mac80211/mlme.c	2020-06-15 23:59:31.611437843 +0100
@@ -1340,7 +1340,7 @@
 		WARN_ON_ONCE(1);
 		/* fall through */
 	case NL80211_BAND_2GHZ:
-	case IEEE80211_BAND_60GHZ:
+	case NL80211_BAND_60GHZ:
 		chan_increment = 1;
 		break;
 	case NL80211_BAND_5GHZ:

